unstructured/file_utils/file_conversion.py

```diff
@@ -54,11 +55,12 @@ def convert_file_to_html_text_using_pandoc(
     exactly_one(filename=filename, file=file)
 
     if file is not None:
-        with tempfile.NamedTemporaryFile() as tmp:
-            tmp.write(file.read())
-            tmp.flush()
+        with tempfile.TemporaryDirectory() as temp_dir_path:
+            tmp_file_path = os.path.join(temp_dir_path, "tmp_file")
```

Comment:
It would be nice if we could do a little better than "tmp_file" as a name. Not sure what `source_format` is, but if it's an extension or something acceptable as one, maybe `f"document.{source_format}"` to produce a name like `document.doc` or `document.epub` or similar if we happen to be looking around in there during debugging or something.